config/24.7/src.conf

```diff
@@ -20,7 +20,7 @@ WITHOUT_NLS=yes
 WITHOUT_NLS_CATALOGS=yes
 WITHOUT_NS_CACHING=yes
 WITHOUT_NTP=yes
-WITHOUT_OFED=yes
+WITH_OFED=yes
```

Comment:
I reckon this is the default so it can be omitted? ;)

Reply:
It is the default on OpenBSD as I understand, but I'm happy to change that if you'd prefer.

Reply:
I've pushed another change after this one to remove

Reply:
thanks, according to src.conf this is indeed the default on amd64.